src/popup/components/Header.vue

```diff
@@ -87,6 +104,9 @@ export default defineComponent({
   },
   setup(props, { root }) {
     const isLoggedIn = useGetter('isLoggedIn');
+    const nodeStatus = useState('nodeStatus');
+    const activeNetwork = useGetter<INetwork>('activeNetwork');
+    const showActiveNetwork = computed<boolean>(() => root.$route.name === ROUTE_ACCOUNT);
```

Comment:
Can be replaced with `showHeaderNavigation` located below.

Reply:
`nodeStatus `  is needed to decide what color state should be used for 
![07DCC4A2-C54B-432E-BD52-5D13E9097885](https://user-images.githubusercontent.com/10424827/208895094-d1d3d7b5-187b-415c-972f-b84459c1993e.png)

`activeNetwork ` needed to display which network is active 

`showActiveNetwork ` is to only display that block when we're on the dashboard screen

Reply:
I'm referring only to `showActiveNetwork`. In my opinion it's obsolete as we should not base on the route name, but on the route meta option.